simba_ml/prediction/time_series/data_loader/transfer_learning_data_loader.py

```diff
@@ -86,8 +86,21 @@ def X_test(self) -> npt.NDArray[np.float64]:
         """
         if self.__X_test is None:
             self.prepare_data()
-            return self.X_test
-        return self.__X_test
+        if self.config.export_path is not None:
+            if not os.path.exists(os.path.join(os.getcwd(), self.config.export_path)):
+                os.mkdir(os.path.join(os.getcwd(), self.config.export_path))
+            assert self.__X_test is not None
+            for i in range(self.__X_test.shape[0]):
+                pd.DataFrame(
+                    self.__X_test[i, :, :],
+                    columns=self.config.time_series.input_features,
+                ).to_csv(
+                    os.path.join(
+                        os.getcwd(), self.config.export_path, f"input_{i}.csv"
+                    ),
+                    index=False,
+                )
```

Comment:
I would like this to be an own function.

Reply:
Changed accordingly. Please look whether this comment is still relevant.